infra/prod/postgres/init.sql

```diff
@@ -0,0 +1,10 @@
+SELECT 'CREATE DATABASE "default"'
```

Comment:
it feels weird to have it duplicated here but also it could be different in prod vs in dev so it kind of makes sense